text/0132-ufcs.md

```diff
@@ -4,10 +4,14 @@
 
 # Summary
 
-This RFC describes a variety of extensions to allow any method to be
-used as first-class functions. The same extensions also allow for
-trait methods without receivers to be invoked in a more natural
-fashion.
+This RFC describes the fully-qualified syntax for making function calls,
+which was previously called Unambiguous or Universal Function Call Syntax
+(UFCS). The [Rust Reference section on call expressions] provides
+an overview of this syntax, and reasons for the change in terminology.
+
+There are a variety of extensions to allow any method to be used as
+first-class functions. The same extensions also allow for trait methods
+without receivers to be invoked in a more natural fashion.
```

Comment:
Changing the structure of this entire paragraph doesn't seem necessary to add the explanatory note, and usually RFCs are preserved as close to as-written as reasonable. The intent is good overall with this PR but please favor additive over subtractive changes.